### ISSUE TITLE
Request for instructions and guide for installing on Unraid

### ISSUE THREAD

**Issue by blitzio** [dmunozv04/iSponsorBlockTV#152]
Hi, please can I request help for a guide on installing iSponsorBlockTV on the Unraid OS? 

I tried following the current docs but find it a little too advanced for someone like me and would appreciate if someone could help out step by step within the context of setting it up on Unraid if possible. 

Thanks for the kind consideration.

**Comment by billxsand** [comment]
Following

**Comment by hardaysknight** [comment]
I used the terminal console in Unraid, then followed the steps outlined in the wiki for using the text based CLI configurator tool.

**Comment by blitzio** [comment]
> I used the terminal console in Unraid, then followed the steps outlined in the wiki for using the text based CLI configurator tool.

Thanks for your reply, and I'm wondering if using the terminal console within Unraid itself, will it also eventually add it to the list of Docker containers so that we can go back to and modify/edit or remove later on? 

For example will it be added to the list of cointainers and viewable here on Unraid:

![image](https://github.com/dmunozv04/iSponsorBlockTV/assets/15014469/2bb4ae39-5adb-4948-965c-a9d7d60e3ae7)

**Comment by hardaysknight** [comment]
Yes it’ll be added to the list of containers. I can see and edit the template from that list.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by blitzio** [comment]
So I managed to get it running via the Unraid terminal using the instructions provided in Installation Wiki, and it seems to finally be working but I noticed that in my Docker Containers list, there seems to be randomized containers being created every few hours and they have weird names. 

Will it just keep multiplying over time? I've had it running for almost a day now and they keep getting generated, see below screenshots.

![image](https://github.com/dmunozv04/iSponsorBlockTV/assets/15014469/036f155a-ae39-4747-a17f-c6ea0558880c)
![image](https://github.com/dmunozv04/iSponsorBlockTV/assets/15014469/6924990e-fda7-43d9-9fd5-0cb16b67ec9e)
![image](https://github.com/dmunozv04/iSponsorBlockTV/assets/15014469/abda47d5-8cf7-4d23-8879-fbbdec9af719)

Could anyone please advise?

Thanks

**Comment by hardaysknight** [comment]
I’d suggest starting over and using the docker hub version when you search through the Unraid Apps page. I guess it just got added.

**Comment by billxsand** [comment]
Update - That fork has been discontinued lol

**Comment by billxsand** [comment]
Got it

Step 1. create a folder for the appdata, I did ' /mnt/user/appdata/isponsorblocktv'
Step 2. Go to the CLI in the docker webui (upper right corner)
Step 3. run  

```
docker run --rm -it \
-v /mnt/user/appdata/isponsorblocktv:/app/data \
ghcr.io/dmunozv04/isponsorblocktv \
--setup-cli
```

it should give an error that there is no config file and that it is creating the config file

then I ran the same command again and it let me set up the config file

**Comment by blitzio** [comment]
@billxsand sorry to resurrect this after so long, your earlier comment was very helpful

could you please advise the command on how to easily update new versions on unraid? 🙏 ty

**Comment by blitzio** [comment]
Ok I think I figured it out on my own with the help of chatgpt and just set up a custom Unraid template for easier maintenance and updates. Sharing the steps I learned for future reference in case someone else needs help

Step 1: Use Unraid GUI to Create the Template
Go to your Docker tab in Unraid

Click "Add Container"

Step 2: Fill out the fields like this:

![Image](https://github.com/user-attachments/assets/46749926-8fea-498d-a416-f384139a8f1f)

i think that's it for now, will wait and see for the next update if it works
